core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -53,6 +54,32 @@ oppia.controller('ExplorationHistory', [
     }
   });
 
+  $scope.submitVersion = function($event, versionNumber) {
+    var checkbox = $event.target;
+    var id = $scope.versionArr.indexOf(versionNumber);
+    if (checkbox.checked && id === -1) {
+      $scope.versionArr.push(versionNumber);
+    }
+    if (!checkbox.checked && id !== -1) {
+      $scope.versionArr.splice(id, 1);
+    }
+    
+    if($scope.versionArr.length == 2) {
+      $scope.versionCountPrompt = "";
+      $scope.changeCompareVersion(versionNumber);
+    }
+    else {
+      $scope.hideHistoryGraph = true;
+      $scope.hideCompareVersionsButton = false;
+
+      if($scope.versionArr.length == 0)
```

Comment:
One note (here and below): please always enclose the bodies of if/else blocks with parentheses. This avoids future mistakes like

   if (...)
       statement1;
       statement2;

(where the developer intends statement2 to be executed within the if block, but in practice it actually gets executed all the time).